media-video/gxine/gxine-0.5.909-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
I see @jsmolic is still stuck in 2020 :grinning:

Reply:
I love 2020 so much :smile: